File: EmbyStat.Services/Models/Music/MusicStatistics.cs
using System;
using System.Collections.Generic;
using System.Text;
using EmbyStat.Services.Models.Stat;

namespace EmbyStat.Services.Models.Music
{
    public class MusicStatistics
    {
        public MusicGeneral General { get; set; }
        public MusicCharts Charts { get; set; }
        public PersonStats People { get; set; }
    }
}

File: EmbyStat.Web/ClientApp/src/app/shared/models/music/music-statistics.ts
import { PersonStatistics } from '../common/person-statistics';
import { GeneralMusicStatistics } from './general-music-statistics';
import { MusicCharts } from './music-charts';

export class MusicStatistics {
    general: GeneralMusicStatistics;
    charts: MusicCharts;
    people: PersonStatistics;
}
